components/Forms/Radio.tsx

```diff
@@ -43,6 +44,9 @@ export const Radio: React.VFC<InputProps> = ({
             className="mb-1.5 text-content font-bold question-link"
             dangerouslySetInnerHTML={{ __html: label }}
           />
+          <p className="ds-inline ds-text-error-border-red ds-text-xl ds-font-medium pl-2">
```

Comment:
If inside the label tag maybe a span is better for this??

Reply:
Good idea.